doc/source/rllib/package_ref/learner.rst

```diff
@@ -91,7 +91,7 @@ Computing Losses
     :nosignatures:
     :toctree: doc/
 
-    Learner.compute_loss
+    Learner.compute_losses
```

Comment:
I changed this API name to be even more clear. This method computes one(!) loss per RLModule (in a MultiRLModule) inside the Learner.

Got rid of the confusing `TOTAL_LOSS` key. We compute this now in the default implementation of `compute_gradients`.